http-server/src/main/java/io/strimzi/http/server/RestService.java

```diff
@@ -0,0 +1,15 @@
+package io.strimzi.http.server;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.ext.web.Router;
+import org.javatuples.Triplet;
+
+/**
+ * The RestService interface is used to identify modules that wish to expose a set of REST endpoints
+ * on the {@link AdminServer}. The service returns a plugin name, a mount
+ * point and a {@link io.vertx.ext.web.Router} containing the endpoints for the plugin.
+ */
+public interface RestService {
+    Future<Triplet<String, String, Router>> registerRoutes(Vertx vertx);
```

Comment:
* There doesn't seem to be any benefit to using a `Triplet` over using a dedicate type (probably a class) which would allow you a change to better define the contract.
* As mentioned, the name seems to provide little value. And there's nothing stopping two modules from having the same name AFAICS, so it's not even bulletproof for the logging use case.
* By letting the module define its mount point you're making it easier to compose modules into an overall service, since you don't need to provide a mount point at the point of binding, but it's less flexible. I guess I'm OK with that, since it's simpler for now.